src/app/components/OptimoPromos/Lists/index.jsx

```diff
@@ -0,0 +1,21 @@
+import { GEL_SPACING_DBL } from '#app/legacy/gel-foundations/src/spacings';
+import styled from '@emotion/styled';
+
+// Bases
+export const StoryPromoLiBase = styled.li`
+  height: 100%;
+  margin-bottom: ${GEL_SPACING_DBL};
+  &:last-child {
+    margin: 0;
+  }
+`;
+
+export const StoryPromoUlBase = styled.ul`
```

Comment:
I think `PromosWrapper` is a more appropriate name

Reply:
I agree that StoryPromoUlBase is too generic. I will suggest using `PromoList` however. I would avoid using the wrapper keyword for lists. We should consider using the Keyword List as in my opinion they are not generic and is much easier for us to identify the component that takes care of the list.

Reply:
Yeh `PromoList` is a good name 🙂